src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,27 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String dumpShape;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String name, String color, String dumpShape) {
+        this.dumpShape = dumpShape;
+        this.setColor(color);
+        this.setName(name);
```

Comment:
don't use setters in constructor, to initialize these fields, use constructor of a parent, for such cases + I don't see such setters here in this class, the same for other figures